Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
```

Comment:
this file should not be here.

Reply:
my program disagrees with you taking it it breaks evrything

Reply:
Please remove it. If you get an error, figure out why and fix it.

Reply:
fine